src/dbus.c

```diff
@@ -418,7 +418,11 @@ static void on_name_lost(GDBusConnection *connection,
                          const gchar *name,
                          gpointer user_data)
 {
-        fprintf(stderr, "Name Lost. Is Another notification daemon running?\n");
+        if (dbus_conn)
```

Comment:
Yes. Full ACK. My thought was that this catches another corner case, when `on_name_acquired` did not work properly and this will ensure, that dunst works better. But actually, the program's dying here. 😂